src/store/reducers/team.js

```diff
@@ -184,6 +184,22 @@ export default (state = INITIAL_STATE, action) => {
         isInviting: false,
       };
 
+    case actionTypes.INVITE_MEMBERS_REQUEST:
+      return {
+        ...state,
+        isInviting: true,
+      };
+    case actionTypes.INVITE_MEMBERS_SUCCESS:
+      return {
+        ...state,
+        isInviting: false,
+      };
+    case actionTypes.INVITE_MEMBERS_FAIL:
+      return {
+        ...state,
+        isInviting: false,
+      };
+
```

Comment:
can we make it generic instead of 3 cases, also last 2 have same state updates.

Reply:
I don't think we need to generic because we need to separate actions.
However we can update state by merging case states into one.

Reply:
Updated.